lib/backbeat/Metrics.js

```diff
@@ -181,13 +181,63 @@ class Metrics {
         });
     }
 
+    /**
+     * Get failed replication stats by ops count and size in bytes
+     * @param {object} details - route details from lib/backbeat/routes.js
+     * @param {function} cb - callback(error, data)
+     * @param {array} data - optional field providing already fetched data in
+     *   order of dataPoints mentioned for each route in lib/backbeat/routes.js
+     * @return {undefined}
+     */
+    getFailedMetrics(details, cb, data) {
+        this._getData(details, data, (err, res) => {
+            if (err && err.type) {
+                this._logger.error('error getting metric: failures', {
+                    origin: err.emthod,
+                    method: 'Metrics.getFailedMetrics',
+                });
+                return cb(err.type.customizeDescription(err.message));
```

Comment:
Having a special error case for this looks a little bit far-fetched to me, but not critical.